addons/test_website_modules/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
```

Comment:
@JKE-be please read the commit message of the commit introducing the test and tell me your opinion 
As discussed, either this test is inside `website` in `@standalone` and run in nightly only (unless it was changed and now tested in staging but that would still not be perfect) which means those tests are useless -> if a breaking change break a standalone test, it will be spotted after merge, then we both know who will need to fix the code "while the test was deactivated".

Or we use a new module that depends of other modules like I did here, even maybe in enterprise to also depend of website_helpdesk etc.